### ISSUE TITLE
Bug: 'availableLangs' is not specified when setting a project with transloco webpack loader

### ISSUE THREAD

**Issue by t-mish** [ngneat/transloco#446]
### Current behavior
If you add transloco to your project and specify the webpack loader, then your project's app.module will be modified by transloco **without** specifiying the available languages to your project.

### Expected behavior
If you add transloco to your project and specify the webpack loader, then your project's app.module will be modified by transloco **with** specifiying the available languages to your project. 

Since transloco suggests you to choose **en** and **es** as default languages, it should also add this languages as a value of **availableLanguages** in the app.module. It shouldn't leave this field blank.

### What's the problem?
In the official docs there is no word that **availableLangs** parameter is required. And transloco doesn't add this parameter to the generated provider config in app.module.

So, if you are newbie and follow the official guide, you won't get you app work as it should. Because you need to know that you must specify the available languages in your config. May be that's sound obvious in retrospective, but for me 3 hours ago it was completely unclear. I thought, that if you already specified the default languge (defaultLangue field) and lib suggested you to create the default languages, then it should count them available too.

### Minimal reproduction of the problem with instructions

Create default project with transloco. **Use webpack loader instead of http.**

### Environment
<pre>
- Transloco version: 2.20.1
- Angular version: 12.0.0
</pre>

### Contribution
I would like to make a pull request for this feature:
[ ] Yes! 🚀
[x] Maybe next time

**Comment by shaharkazaz** [comment]
@t-mish You were prompt with the question of which loader to use in v2.20.1?

**Comment by t-mish** [comment]
@shaharkazaz Thanks for the reply. Yes, I was. Your assumption sounds like a true. 

Before I realized what's wrong, everything worked well except you see everywhere something like 'en.hello_world' instead of 'Hello, world'. I started debugging and saw that when you don't specify the 'alailableLangs' it just replaces the passed option object (where this field is included) with the default one (which is empty and hardcoded). For me this is behaviour was unexcpected, because I saw that everything compiled well (even without warnings).
